packages/apollos-church-api/src/data/features/resolver.js

```diff
@@ -0,0 +1,14 @@
+import { Features as baseFeatures } from '@apollosproject/data-connector-rock';
+import { resolverMerge } from '@apollosproject/server-core';
+
+const resolver = {
+  HeaderFeature: {
+    // TODO waiting on core release
+    // sharing: ({ body }) => ({
+    // title: 'Share via...',
+    // message: body,
+    // }),
+  },
+};
+
+export default resolverMerge(resolver, baseFeatures);
```

Comment:
Just a question here: Why no "Notes Feature" in the resolver? It's in the schema and the function for it is in the data-source so ...

Is it not needed? Or just forgotten?